.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: node_js
+node_js:
+   - '4.5'
+before_script:
+   - npm install -g grunt
+script:
+   - grunt
```

Comment:
Please make this more specific, e.g. `grunt standards`, so that it's easier to tell what's happening on Travis